src/environment-variables/index.js

```diff
@@ -53,8 +54,14 @@ function getBoolean(value) {
   return !!value;
 }
 
+function getCompleteEpisodePath(){
```

Comment:
Thanks for the contribution. Most of the changes look clean, but I would suggest you to fix the eslint and prettier warnings.

For example, my IDE(IntelliJ) shows these eslint and prettier issues:
![Screenshot_20230807_115446](https://github.com/Schrodinger-Hat/youtube-to-anchorfm/assets/44746117/20565bef-e21c-461e-a3c0-d383bd61b8c8)

Could you fix them?

Reply:
Do it shows the best correction?

Reply:
fixed in a new commit